TRANSLATE.md

```diff
@@ -56,6 +56,9 @@ At some occasions you'll find a variable like `$INDS`. Leave them as is, they wi
 
 Regarding the campaign name *Public Money, Public Code*. In the past we haven't made good experiences with translating such campaign names. All our graphics, logos, and other communication is using this brand. So if you can, just stick to the English term.
 
+### static/js/
+
+Here there is a file called `onScrollMenu.js` which contains the abbreviations of all the languages used in an array, on the 3rd line. Add the abbreviation of the language you've added to this array.
```

Comment:
@Remorax can you just add that these abbreviations are the languageCode mentioned in the toml files in site/languages

Reply:
Sure :+1: